landing/index.html

```diff
@@ -4,52 +4,29 @@
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <!--
-      The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags
-    -->
-    <meta
-      name="description"
-      content="End Of Coffee - Track your office inventory"
-    />
+    <meta name="description" content="End Of Coffee" />
```

Comment:
proposal: the description is important in SEO and should be more descriptive: "End of coffee helps you track all the stuff you care about!"